frontend/src/components/scheduleEntry/ScheduleEntries.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <div>
+    <slot
+      :scheduleEntries="scheduleEntries"
+      :loading="apiScheduleEntries._meta.loading"
+      :showActivityCreateDialog="showActivityCreateDialog"
+      :showActivityEditDialog="showActivityEditDialog" />
+    <dialog-activity-create
+      ref="dialogActivityCreate"
+      :schedule-entry="popupEntry"
+      @activityCreated="afterCreateActivity($event)"
+      @creationCanceled="cancelNewActivity" />
+    <dialog-activity-edit
+      ref="dialogActivityEdit"
+      :schedule-entry="popupEntry" />
+    <v-btn
+      v-if="showButton"
+      :fixed="$vuetify.breakpoint.xs"
+      :absolute="!$vuetify.breakpoint.xs"
+      dark
+      fab
+      style="z-index: 3"
+      bottom
+      right
+      class="fab--bottom_nav float-right"
+      color="red"
+      @click.stop="createNewActivity()">
+      <v-icon>mdi-plus</v-icon>
+    </v-btn>
+  </div>
+</template>
+
+<script>
+import DialogActivityCreate from '@/components/dialog/DialogActivityCreate'
+import DialogActivityEdit from '@/components/dialog/DialogActivityEdit'
+import { defineHelpers } from '@/components/scheduleEntry/dateHelperLocal'
+
+export default {
+  name: 'ScheduleEntries',
+  components: {
+    DialogActivityCreate,
+    DialogActivityEdit
+  },
+  props: {
+    period: { type: Function, required: true },
+    showButton: { type: Boolean, required: true }
+  },
+  data () {
+    return {
+      scheduleEntries: [],
+      deleteTempEntryCallback: () => {},
+      popupEntry: {}
+    }
+  },
+  computed: {
+    apiScheduleEntries () {
+      // TODO for SideBar, add filtering for the current day when backend supports it
+      return this.period().scheduleEntries()
```

Comment:
Note to self, create a new issue for this after this PR is merged. This is now possible and should be another good first issue.

Reply:
#680